src/analysis/normed_space/pi_Lp.lean

```diff
@@ -4,25 +4,29 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Sébastien Gouëzel
 -/
 import analysis.mean_inequalities
+import data.fintype.order
 
 /-!
 # `L^p` distance on finite products of metric spaces
 Given finitely many metric spaces, one can put the max distance on their product, but there is also
-a whole family of natural distances, indexed by a real parameter `p ∈ [1, ∞)`, that also induce
-the product topology. We define them in this file. The distance on `Π i, α i` is given by
+a whole family of natural distances, indexed by a parameter `p : ℝ≥0∞`, that also induce
```

Comment:
No, I think it works for any `p`. Two quasi-metrics generate the same topology if they are equivalent (in the standard sense of metric equivalence), and all the `p` quasi-metrics in `ℝⁿ` are equivalent because the balls can be scaled to fit inside each other.